docs/LearningOutcomes.adoc

```diff
@@ -144,7 +143,7 @@ e.g. `Main.VERSION`, `Name.EXAMPLE`, `Utils.isAnyNull(...)`
 
 === References
 
-* https://se-edu.github.io/se-book/oopDesign/classes/classLevelMembers/[se-edu/se-book: Design: OOP: Classes: Class-Level Members]
+* https://se-edu.github.io/se-book/oop/classes/classLevelMembers/[se-edu/se-book: Design: OOP: Classes: Class-Level Members]
```

Comment:
Can add this one too https://se-edu.github.io/se-book/cppToJava/classes/classLevelMembers/

Reply:
Also add this to the exceptions LO.
https://se-edu.github.io/se-book/cppToJava/exceptions/